### PR TITLE
Localize location names

### PR DESCRIPTION
**Pull request by tahmidrahman-dsi**
In 

- record audit history
- organization list
- user list page
- location picker of the user list 
- field agent registration office select

### CODE REVIEW

File: packages/client/src/utils/locationUtils.ts [hunk 1]
```diff
@@ -129,7 +130,7 @@ export function generateSearchableLocations(
 
     return {
       id: location.id,
-      searchableText: location.name,
+      searchableText: getLocalizedLocationName(intl, location),
       displayLabel: locationName
```
Comment (Zangetsu101) [inline packages/client/src/utils/locationUtils.ts:134]: I wonder which one of these 2 should be localized or both?

Comment (tahmidrahman-dsi) [reply packages/client/src/utils/locationUtils.ts:134]: well, when the labels are localized, wouldn't the user be searching by the localized names? 🤔

Comment (Zangetsu101) [reply packages/client/src/utils/locationUtils.ts:134]: makes sense. then what about the displayLabel? Isn't that what's actually visible to the user?

Comment (Zangetsu101) [reply packages/client/src/utils/locationUtils.ts:134]: ah okay it's already localized by the `generateLocationName` function right?

**Review by Zangetsu101 — APPROVED** [review]

File: packages/client/src/utils/locationUtils.ts (hunk 1 above)
Comment (tahmidrahman-dsi) [reply packages/client/src/utils/locationUtils.ts:134]: yes, already localized

**Pull request merged**